### PR TITLE
beta guide: add a new chapter on file watching

### PR DESCRIPTION
**Pull request by oligot**
This is the markdown version as discussed here
https://groups.google.com/forum/#!topic/jspm-io/4Af1T6GBvkk

It's probably a good idea to review it before publishing as English is
not my mother tongue

### CODE REVIEW

File: 0.17-beta-guide/file-watching.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+jspm loads and transpiles everything in the browser, and in development this workflow works really well.
+However, when your app reaches a certain size the overhead of running everything in the browser leads to your app taking multiple seconds to refresh.
+
+At this point, you can switch to bundling your application whenever you save a file to generate a large, unminified bundle file which you can give to your browser.
+This is made possible thanks to recent additions to jspm that enable builds to make use of a build cache.
+
+An alternative to this approach is to use hot-reloading whis is covered in the next chapter.
+
+To start watching files, we will use the `jspm bundle` command with the new `watch` flag
+> jspm bundle test.js dev-bundle.js --watch
```
Comment (guybedford) [inline 0.17-beta-guide/file-watching.md]: I think the example should be based on `--dev --inject` flags.

**Comment by guybedford** [comment]
:+1: I'll edit this in when I can.

**Comment by oligot** [comment]
Ok, great

**Comment by guybedford** [comment]
Thanks, I've merged this into the beta guide with some adjustments.

**Pull request closed without merging**